libpromises/systype.c

```diff
@@ -95,6 +97,7 @@ const char *const VPSOPTS[] =
     [PLATFORM_CONTEXT_AIX] =  "-N -eo user,pid,ppid,pgid,pcpu,pmem,vsz,ni,stat,st=STIME,time,args",       /* aix */
     /* Note: keep in sync with GetProcessOptions()'s hack for Linux 2.4 */
     [PLATFORM_CONTEXT_LINUX] = "-eo user,pid,ppid,pgid,pcpu,pmem,vsz,ni,rss:9,nlwp,stime,etime,time,args",/* linux */
+    [PLATFORM_CONTEXT_BUSYBOX] = "",                  /* linux / busybox */
```

Comment:
If I recall, it just needs the "w" argument to avoid truncation. I'll login to our router to find out. :-p